app/controllers/sessions_controller.rb

```diff
@@ -4,11 +4,18 @@ def new; end
   def create
     @user = User.find_by email: params[:session][:email].downcase
     if @user&.authenticate(params[:session][:password]).present?
-      flash[:success] = t "login_success"
-      login @user
-      params[:session][:remember_me] ==
-        Settings.checkbox ? remember(@user) : forget(@user)
-      redirect_back_or @user
+      if @user.activated?
+        log_in @user
+        if params[:session][:remember_me] == Settings.checkbox
```

Comment:
toán tử 3 ngôi đoạn này dc k e

Reply:
dùng được ạ nhưng mà em chạy rubocop nó bảo toán tử 3 ngôi mà dài quá thì chuyển về if else ạ